changed: minor improvements to NL translation in Grammar, spelling, a…

Hobbesball opened the pull request:
…nd correct variable init.

Main changes:
- Capitalised words only if they were capitalised in English version
- removed space on line 20 so {%0} variable would be initialised correctly
- improved grammar and spelling (e.g. 'probeeren'->proberen, modern spelling e.g. 'Kwast naam'-> kwastnaam)

Could you please label this PR with the label 'hacktoberfest-approved'? That way we're sure this will count towards my hacktoberfest PR's ;) Thanks <3 

referencing #130

Hobbesball committed:
changed: minor improvements to NL translation in Grammar, spelling, and correct variable init

thebigsmileXD commented:
Oh, `hacktoberfest-approved`? looks like they've changed things, thy done

thebigsmileXD commented on resources/lang/nld.ini:
```diff
@@ -17,13 +17,13 @@ error.limitexceeded = "U probeert te veel blokken tegelijk te bewerken. Verklein
-error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {% 0} te gebruiken"
+error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {%0} te gebruiken"
 error.noclipboard = "Geen klembord gevonden - maak eerst een klembord"
 warning.differentlevel = "[WAARSCHUWING] Je bewerkt een wereld waar je momenteel niet in zit!"
```
Oh btw this changed to .differentworld recently, mind patching?

Hobbesball replied:
I've changed .nosession to .differentworld, is that what you intended?

Aericio replied:
should've been more specific, `differentlevel` -> `differentworld`

Hobbesball committed:
changed: .nosession to .differentworld

Hobbesball commented:
> Oh, `hacktoberfest-approved`? looks like they've changed things, thy done

hi, sorry I meant 'hacktoberfest-accepted', not approved, that's my bad! Could you change the label to hacktoberfest-accepted before merging?

Aericio commented on resources/lang/nld.ini:
```diff
@@ -75,14 +75,14 @@ tool.wand.lore.1 = "Klik met de linkermuisknop op een blok om de positie 1 van e
 tool.wand.lore.3 = "Gebruik //togglewand om van functionaliteit te wisselen"
 tool.wand.disabled = "De toverstaf is uitgeschakeld. Gebruik //togglewand om het opnieuw in te schakelen"
-tool.wand.setenabled = "Het gereedschap Toverstaf is nu{ %0}!"
+tool.wand.setenabled = "De toverstaf is nu{ %0}!"
```
```suggestion
tool.wand.setenabled = "De toverstaf is nu {%0}!"
```

Aericio commented:
> Oh, hacktoberfest-approved? looks like they've changed things, thy done

Maybe it was because of the `Amazing Project` PR spam that was happening.

Hobbesball committed:
Update resources/lang/nld.ini

Co-authored-by: Aericio <16523741+Aericio@users.noreply.github.com>

Hobbesball committed:
Update resources/lang/nld.ini

Co-authored-by: Aericio <16523741+Aericio@users.noreply.github.com>

Hobbesball committed:
Update resources/lang/nld.ini

Co-authored-by: Aericio <16523741+Aericio@users.noreply.github.com>

thebigsmileXD commented:
.nosession and .differentworld, something looks like it got lost

inxomnyaa committed:
Change capitalisation of "Commands"

This is always a standalone word
`-=+=- Commands -=+=-`
(full list of commands here)

inxomnyaa committed:
Fix date
